scrapy/settings/default_settings.py

```diff
@@ -158,6 +158,8 @@
 }
 FEED_EXPORT_INDENT = 0
 
+FEED_STORAGE_S3_ACL = None
```

Comment:
I'm not 100% sure it is the way to go, but I've suggested using empty strings instead of None in such cases (see https://github.com/scrapy/scrapy/pull/3199#discussion_r181327270), and there is code which does that.

Reply:
I've just read the comment you linked here. I understand that `''` or `None` would produce the same outcome because both values will eval to `False`.

Anyway, I have just replaced its default value to `''` (empty string) to make it consistent with other similar settings.